examples/neighbors/plot_lof.py

```diff
@@ -0,0 +1,51 @@
+"""
+=================================================
+Anomaly detection with Local Outlier Factor (LOF)
+=================================================
+
+This example uses the LocalOutlierFactor (LOF) estimator
+for anomaly detection.
+LOF is a unsupervised Outlier Detection method.
+Its anomaly score measures the local deviation of density of a given
+sample with respect to its neighbors.
+By comparing the local density of a sample to the local densities of
+its neighbors, one can identify samples that have a substantially lower
+density than their neighbors. These are considered as outliers.
```

Comment:
I will more say : 
"This example presents the Local Outlier Factor (LOF) estimator to the anomaly detection. The LOF is an unsupervised outlier detection method which compares the local density deviation of a given sample with respect to its neighbors. It considers as outlier samples that have a substantially lower density than their neighbors."